src/main/webapp/app/exercises/shared/exercise-headers/header-exercise-page-with-details.component.html

```diff
@@ -1,116 +1,142 @@
+                <span>
+                    {{ 'artemisApp.courseOverview.exerciseDetails.assessmentType' | artemisTranslate }}
+                    {{ exercise.assessmentType }}
+                </span>
+            </h4>
+            <h4 class="d-md-flex" *ngIf="exercise.categories || exercise.difficulty">
+                <span *ngFor="let category of exerciseCategories">
+                    <span class="badge text-white me-1" [ngStyle]="{ backgroundColor: category.color }">{{ category.category }}</span>
+                </span>
+                <span *ngIf="exercise.difficulty">
+                    <jhi-difficulty-badge class="me-1" [exercise]="exercise" [showNoLevel]="false"></jhi-difficulty-badge>
+                </span>
+            </h4>
+            <h4 class="row" *ngIf="submissionPolicy">
+                <span>
+                    {{ 'artemisApp.programmingExercise.submissionPolicy.submissionLimitTitle' | artemisTranslate }}:
+                    {{
+                        submissionPolicy.submissionLimit +
+                            (submissionPolicy.exceedingPenalty
+                                ? ' (' +
+                                  ('artemisApp.programmingExercise.submissionPolicy.submissionPenalty.penaltyInfoLabel' | artemisTranslate) +
+                                  ': ' +
+                                  submissionPolicy.exceedingPenalty +
+                                  'P' +
+                                  ')'
+                                : '')
+                    }}
+                    <fa-icon
+                        [icon]="faQuestionCircle"
+                        ngbTooltip="{{ 'artemisApp.programmingExercise.submissionPolicy.submissionPolicyType.' + submissionPolicy.type + '.tooltip' | artemisTranslate }}"
+                        placement="top"
+                    ></fa-icon>
+                </span>
+            </h4>
+        </div>
+        <div class="col-auto ms-md-auto">
             <ng-template [ngIf]="isExamMode" [ngIfElse]="courseMode">
-                <h4>
-                    <span class="me-2" *ngIf="!isTestRun"> {{ 'artemisApp.exam.endDate' | artemisTranslate }} : {{ exam?.endDate | artemisDate }} </span>
-                    <span *ngIf="exam?.endDate && !isTestRun" [ngClass]="exerciseStatusBadge" class="badge">
-                        {{ exam?.endDate | artemisTimeAgo }}
-                    </span>
-                </h4>
-                <h4 *ngIf="!isTestRun">
-                    <span class="me-2"> {{ 'artemisApp.exam.publishResultsDate' | artemisTranslate }} : {{ exam?.publishResultsDate | artemisDate }} </span>
-                    <span *ngIf="exam?.publishResultsDate" [ngClass]="exerciseStatusBadge" class="badge">
-                        {{ exam?.publishResultsDate | artemisTimeAgo }}
-                    </span>
-                </h4>
+                <ng-container *ngIf="!isTestRun">
+                    <h5 class="{{ isNextDueDate[0] ? 'h3' : '' }}">
```

Comment:
```suggestion
                    <h5 [class.h3]="isNextDueDate[0]">
```
This should also work :thinking: (also in other cases)